lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,5 @@
+module RakeArguments
+  def self.flags
+    ARGV.drop_while { |e| e != '--' }.drop(1)
```

Comment:
I definitely agree that the module is a bit overkill now, but I'm a little unsure of what the better alternative is. Right now the `flags` method is used in 2 places in the Rakefile (dev tests and exercise tests). Should I duplicate that in both places?

I'm not able to define a `flags` method in the Rakefile.